compass/ocean/tests/parabolic_bowl/namelist.forward

```diff
@@ -0,0 +1,13 @@
+config_run_duration='0003_00:00:00'
+config_time_integrator='RK4'
+config_vert_coord_movement='impermeable_interfaces'
+config_ALE_thickness_proportionality='weights_only'
+config_use_wetting_drying=.true.
+config_prevent_drying=.true.
+config_drying_min_cell_height=2.0e-2
+config_zero_drying_velocity=.true.
+config_verify_not_dry=.true.
+config_thickness_flux_type='upwind'
+config_use_cvmix=.false.
+config_use_implicit_bottom_drag=.false.
```

Comment:
```suggestion
config_bottom_drag_mode='explicit'
```

Reply:
Is this right? Or is drag supposed to be entirely off?

Reply:
Thanks for catching this. Bottom drag should be off for this case, so I deleted this and added `config_implicit_constant_bottom_drag_coeff = 0.0`